changelog/front-commerce-3.9-2.34.mdx

```diff
@@ -0,0 +1,379 @@
+            Header: new URL("./components/AcmeHeader.tsx", import.meta.url),
+            Footer: new URL("./components/AcmeFooter.tsx", import.meta.url),
+          },
+        },
+      });
+
+      // Another example with a feature flag that can be toggled on/off
+      hooks.registerFeature("newsletter", {
+        flags: {
+          enabled: false,
+        },
+      });
+    },
+  },
+});
+```
+
+This is just a quick overview of the new Features API. We hope you'll find it
+useful in your projects. If you have any questions or need help, don't hesitate
+to reach out to us!
+
+</details>
+
+These first use cases showcase the potential of the Features API. **Expect more
+extensive usage in future versions as we continue to explore the
+possibilities!**
+
+## Other changes
+
+We additionally worked on changes and bug fixes across different areas of the
+product. You'll find a summary of these changes below.
+
+```mdx-code-block
+<details>
+  <summary><h3 className="mb-0">Features</h3></summary>
+```
+
+- **Theme:**
+
+  - we improved the **default display of shipping methods** to avoid text
+    overflowing outside of their container elements, and to avoid breaking words
+    in a way that reduces readability.
+  - dynamic facets are now more generic and can be extended to support more
+    types of filters display. We added support for two new types of filters:
+    `TextSearchFacet` and `RangeSearchFacet`. Read more in the
+    [Facets feature documentation](/docs/3.x/extensions/theme-chocolatine/reference/features#facets).
+  - order details page now displays the image of ordered products
+  - registration form now supports company registration if the platform supports
+    it
+  - it is now possible to toggle account registration for individuals and
+    companies using configurations. The registration page will display a message
+    and a button to contact the merchant if account registration is disabled
+  - user information screens now support phone number for platforms that require
+    it
+  - the theme now relies on a platform agnostic `reference` field for product,
+    cart items and orders to support platforms where a SKU is not relevelant
+  - promotional prices are now displayed on each cart item line.
+
+- **Service worker:**
+  - we tackled a long-standing feature request from our customers to reduce
+    service-worker logs verbosity. We now allow to configure the disablement of
+    service-worker logs in `entry.worker.ts` using the `disableDevLogs` and
+    `disableStorefrontContentDevLogs` options. **We changed the default behavior
+    and disabled Workbox dev logs** but maintained Storefront Content ones.
+- **Storybook**
+  - public configuration are now available in Storybook based on the definition
+    in `front-commerce.config.ts`
+  - we added new stories for several components (related to price display
+    configuration). Browse
+    [our demo's styleguide](https://magento2.front-commerce.app/styleguide) to
```

Comment:
I think we would need to deploy the styleguide on production.

Currently we have [devOnly](https://gitlab.blackswift.cloud/front-commerce/cloud/front-commerce-deployments/front-commerce-v3/front-commerce-v3-core/-/settings/ci_cd) set in our environments for deployments.

Reply:
Agreed. @Kagaminara could you do it as part of the demos update please?

Reply:
Done!